ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx

```diff
@@ -137,6 +139,35 @@ enum Mode {
   BadInput = 'bad-input',
 }
 
+const handleMetacardUpdate = ({
+  lazyResult,
+  attributes,
+  onComplete,
+}: {
+  lazyResult: LazyQueryResult
+  attributes: MetacardAttribute[]
+  onComplete: () => void
+}) => {
+  const payload = [
+    {
+      ids: [lazyResult.plain.metacard.properties.id],
+      attributes,
+    },
+  ]
+  setTimeout(() => {
+    $.ajax({
+      url: `./internal/metacards?storeId=${lazyResult.plain.metacard.properties['source-id']}`,
+      type: 'PATCH',
+      data: JSON.stringify(payload),
+      contentType: 'application/json',
+    })
+      .then((response: any) => {
+        ResultUtils.updateResults(lazyResult.getBackbone(), response)
+      })
+      .always(onComplete)
```

Comment:
The one thing I realized is odd (we should make a ticket to go back and address), is that since we do an "always" here it doesn't matter if the update succeeds or not.  We should update this to have an onSuccess, onFailure, onComplete so that we can respond appropriately.  Right now if a failure occurs it'll still say "successfully updated".

![](https://miro.medium.com/max/1274/1*ON_d7DWgW8g8uu3EBntfNw.png)